octoprint_Spoolman/templates/Spoolman_sidebar.jinja2

```diff
@@ -127,7 +130,10 @@
                             <span>[<span data-bind="
                                 text: $data.spoolDisplayData.filament.material.displayShort,
                                 attr: { title: $data.spoolDisplayData.filament.material.displayValue }
-                            "></span>]</span>
+                            "></span>] <span data-bind="
```

Comment:
I'm also wondering - how important it is to display "lot no" its entirety? Potential 64 characters seems really long, and the space seems quite crowded already, with just the filament type, name & brand.

I was thinking, that maybe the lot number could be treated a bit differently - make it truncated (eg. for lot numbers longer than 9 characters, show the first 3 and last 3 characters, with `...` in the middle), and retain the full value in a tooltip, so that it's still accessible when hovered over? It's not that important in the list picker, but here, in the sidebar, it might be worth the effort, as long as it still serves its purpose. What do you think about that?

Reply:
I like this idea, I will explore it some more in a separate PR!